consensus/istanbul/uptime/store/store.go

```diff
@@ -0,0 +1,24 @@
+package store
+
+import (
+	"github.com/ethereum/go-ethereum/consensus/istanbul/uptime"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/ethdb"
+)
+
+type uptimeStoreImpl struct {
+	db ethdb.Database
+}
+
+func New(db ethdb.Database) uptime.Store {
```

Comment:
Since this `New()` is only used inline inside calls to `uptime.NewMonitor()`, would it simplify things to have `NewMonitor()` take in the `db` and call this `New()` itself?  Or maybe consider removing `uptimeStoreImpl` entirely?